components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -40,6 +40,57 @@ UpdateVersion.propTypes = {
     releaseNotesUrl: PropTypes.string,
 };
 
+// Label renders a tag showing a name and a description in a tooltip.
+// If an URL is provided, clicking on the tag will open the URL in a new tab.
+export const Label = ({name, description, url}) => {
+    const tag = (
+        <span className='tag'>
+            {name.toUpperCase()}
+        </span>
+    );
+
+    let label;
+    if (description) {
+        label = (
+            <OverlayTrigger
+                delayShow={Constants.OVERLAY_TIME_DELAY}
+                placement='top'
+                overlay={
+                    <Tooltip id={'plugin-marketplace_label_' + name.toLowerCase() + '-tooltop'}>
+                        {description}
+                    </Tooltip>
+                }
+            >
+                {tag}
+            </OverlayTrigger>
+        );
+    } else {
+        label = tag;
+    }
+
+    if (url) {
+        return (
+            <a
+                aria-label={name.toLowerCase()}
+                className='style--none more-modal__row--link'
+                target='_blank'
+                rel='noopener noreferrer'
+                href={url}
+            >
+                {label}
+            </a>
+        );
+    }
+
+    return label;
+};
+
+Label.propTypes = {
```

Comment:
I know we recognized that the plugin name and description aren't localized, but since we localize the `Download` and `Configure` buttons, I wonder if we shouldn't at least try to support localization for the labels as well.

Strawman:
* Marketplace sends down an id for each label
* Webapp uses `FormattedMessage` to render id, with default text provided by marketplace to allow dynamic labels in English.
* As new labels are introduced, we add them to the next server version to allow for localization.

Thoughts?

Reply:
If a text gets updated on the Marketplace, the webapp should show the new English text for all languages, right? Otherwise the semantic is different in English and in other languages.

If this is correct, I would opt to not include localization support, as it the implementation of this requirement on the webapp is not easy.

Reply:
@lieut-data and I talked about this offline and agreed, that it's not worth to add the complexity that is needed for this change.

The "right" way to add localization is to localize the labels in the marketplace and send the localized string via the server to the client.